IncSet.h

```diff
@@ -15,7 +15,7 @@ in the source distribution for its full text.
 #include "Vector.h"
 
 
-#define INCMODE_MAX 40
+#define INCMODE_MAX 254
```

Comment:
```suggestion
#define INCMODE_MAX 255
```

This has the added bonus of also causing the buffer in the `IncSet` structure to have proper alignment with no additional padding.